### ISSUE TITLE
lando v3.0.0-beta.11 exe fails to open in windows

### ISSUE THREAD

**Issue by nalipaz** [kalabox/lando#310]
Very simple issue, but beta.10 works and beta.11 does not on a clean windows 10 install. It simply does not open when I attempt to double-click.

*windows disclaimer: I am testing this for coworkers* - I feel so dirty reporting windows issues, I can hear the responses already, "yeah right, a coworker, sure..."

BTW, beta.11 worked fine on Debian Jessie

**Comment by brentratliff** [comment]
Asking for a friend, lol.  I had a similar issue on my Windows machine as a primarily Mac user.  Is it the installer that doesn't work or the shortcut the installer creates after installation?  I think that was removed in beta11 but it was there in beta10.  It links to a gui that doesn't work.

**Comment by nalipaz** [comment]
It is the installer itself. Double-clicking the installer does not open the GUI. Whereas on the beta10 the GUI opens up promptly.

**Comment by brentratliff** [comment]
I updated from 10 to 11 last night and it worked on the Creator's Update to Windows 10 Pro.  I guess if it worked with beta10 for you I don't know why it wouldn't work for 11.  I did have to give permission when I got a prompt about it being from an unidentified developer which I don't remember doing for beta10.  If it's a vanilla installation, is Hyper-V enabled?  I'm not a developer on this project, just a Windows user of it.

**Comment by pirog** [comment]
AFAIK we didn't make any changes to the installer between beta10 and beta11 so there is no reason, at least in terms of code changes, that would explain this

**Comment by nalipaz** [comment]
For sure Hyper-V is enabled, otherwise I wouldn't have been able to install beta10. Double-clicking beta10 is no problem, but beta11 does nothing. This is on a machine that had never had lando installed before. I attempted downloading the executable more than once too, to see if maybe I got a bad download, but nothing.

I will try using the executable to install atop the beta10 and see if it goes through, but that might not be until early next week since I need to move onto finishing items up for sprint rather than doing the lando setup stuff for the remainder of the week.
